recipe/meta.yaml

```
@@ -12,18 +12,17 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -v
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - avro
     - isort
     - pip
```

the usual `setuptools` and `wheel` is required
```suggestion
    - pip
    - setuptools
    - wheel
```

you can see that `setuptools` is imported in the `setup.py` here https://inspector.pypi.io/project/avro-python3/1.10.2/packages/cc/97/7a6970380ca8db9139a3cc0b0e3e0dd3e4bc584fb3644e1d06e71e1a55f0/avro-python3-1.10.2.tar.gz/avro-python3-1.10.2/setup.py#line.39

---

`setuptools` and `wheel` in general are very common host dependencies for pure python packages.
It is easier to assume setuptools and wheel are always there, except some cases.

---

@psteyer and this one :)